.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish snapshots to maven
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
can we restrict the branch to 1.3 ?

Reply:
I pushed a commit to restrict this to 1.3. The additional entries are harmless, but don't serve a functional purpose